packages/airnode-protocol/credentials.example.json

```diff
@@ -1,30 +1,116 @@
 {
-  "etherscanApiKey": "",
+  "etherscan": {
+    "apiKey": {
+      "mainnet": "",
+      "ropsten": "",
+      "rinkeby": "",
+      "goerli": "",
+      "kovan": "",
+      "arbitrumOne": "<arbitrum>",
+      "arbitrumTestnet": "<arbitrum-testnet>",
+      "avalanche": "",
+      "avalancheFujiTestnet": "<avalanche-testnet>",
+      "bsc": "",
+      "bscTestnet": "<bsc-testnet>",
+      "opera": "<fantom>",
+      "ftmTestnet": "<fantom-testnet>",
+      "xdai": "<gnosis>",
+      "optimisticEthereum": "<optimism>",
+      "optimisticKovan": "<optimism-testnet>",
+      "polygon": "",
+      "polygonMumbai": "<polygon-testnet>"
+    }
+  },
   "networks": {
     "mainnet": {
       "accounts": { "mnemonic": "" },
       "chainId": 1,
-      "url": ""
+      "url": "https://mainnet.infura.io/v3/8997be2479d94f91a24df8b56a8e98df"
```

Comment:
These URLs are now public and they are valid (not a huge deal though).

Reply:
Yep, it belongs to me and it's free account.

Reply:
For reference :sweat_smile: 
https://github.com/ethers-io/ethers.js/blob/50d712d6afc927aaee68b482157598554a45d403/packages/providers/src.ts/infura-provider.ts#L17

Reply:
lul :D

Reply:
This makes me feel better about past mistakes :P

Reply:
You're too optimistic. It just makes me feel worse about using ethers.js :sweat_smile: